bin/versions.bash

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2034 # Variables are expected to be unused in this file
 # Release from 2024-05-02 which includes nervous_system_parameters.
```

Comment:
Nit (feel free to ignore): you could consider updating the comment above the new aggregator version for consistency.